packages/gasket-plugin-docusaurus/README.md

```diff
@@ -0,0 +1,62 @@
+# @gasket/plugin-docusaurus
+
+Use [Docusaurus] to serve a website view of the collated docs from [@gasket/plugin-docs].
+
+## Installation
+
+#### New apps
+
+```
+gasket create <app-name> --plugins @gasket/plugin-docs,@gasket/plugin-docusaurus
+```
+#### Existing apps
+
+```
+npm i @gasket/plugin-docs @gasket/plugin-docusaurus
+```
+
+Modify `plugins` section of your `gasket.config.js`:
+
+```diff
+module.exports = {
+  plugins: {
+    add: [
++      '@gasket/plugin-docs',
++      '@gasket/plugin-docusaurus'
+    ]
+  }
+}
+```
+
+Now, when you run `npx gasket docs` in your app, a nice Docusaurus website will be
+launched in your browser.
+
+
+## Configuration
+
+To be set under `docusaurus` in the `gasket.config.js`. A required `docusaurus.config.js` will generated in the root directory if one is not present. Docusaurus allows for additional [configuration/customization](https://docusaurus.io/docs/api/docusaurus-config) options and you can define those directly in the `docusaurus.config.js`.
```

Comment:
```suggestion
To be set under `docusaurus` in the `gasket.config.js`.
```

Move the docusarus config file description _after_ your plugin config example, maybe with a h3 heading. On first read I was slightly confused if the config you were walking about was for Gasket or Docusaurus itself.